src/pages/SettingsPage/SettingsPage.tsx

```diff
@@ -0,0 +1,263 @@
+          <OptionRow>
+            <FormControl fullWidth>
+              <InputLabel id="language-select-label">{t.language}</InputLabel>
+              <Select
+                labelId="language-select-label"
+                id={ACCESSIBILITY_IDS.languageSelect}
+                value={preferences.language}
+                label={t.language}
+                // eslint-disable-next-line @typescript-eslint/no-explicit-any
+                onChange={(e) => setLanguage(e.target.value as any)}
+              >
+                {LANGUAGE_OPTIONS.map(({ value, label }) => (
+                  <MenuItem key={value} value={value}>
+                    {label}
+                  </MenuItem>
+                ))}
+              </Select>
+            </FormControl>
+          </OptionRow>
+        </OptionGroup>
+      </SectionCard>
+
+      <SectionCard data-testid={SETTINGS_TEST_IDS.privacySection}>
+        <SectionTitle>{t.sectionPrivacy}</SectionTitle>
+        <PrivacyBox>
+          <PrivacySwitches />
+        </PrivacyBox>
+      </SectionCard>
+    </>
+  );
+
+  const MobileLayout = () => (
+    <>
+      <Accordion defaultExpanded>
+        <AccordionSummary expandIcon={<ExpandMoreIcon />}>
+          <SectionTitle>{t.sectionTheme}</SectionTitle>
+        </AccordionSummary>
+        <AccordionDetails>
+          <OptionRow>
+            <OptionDescription>{t.theme}</OptionDescription>
+            <Switch
+              checked={preferences.theme === 'dark'}
+              onChange={toggleTheme}
+              inputProps={{ 'aria-label': t.theme }}
+              id={ACCESSIBILITY_IDS.themeSwitch}
+            />
+          </OptionRow>
+        </AccordionDetails>
+      </Accordion>
+
+      <Accordion>
+        <AccordionSummary expandIcon={<ExpandMoreIcon />}>
+          <SectionTitle>{t.sectionNotifications}</SectionTitle>
+        </AccordionSummary>
+        <AccordionDetails>
+          <OptionRow>
+            <OptionDescription>{t.notifications}</OptionDescription>
+            <Switch
+              checked={preferences.notifications}
+              onChange={(e) => setNotifications(e.target.checked)}
+              inputProps={{ 'aria-label': t.notifications }}
+              id={ACCESSIBILITY_IDS.notificationsSwitch}
+            />
+          </OptionRow>
+        </AccordionDetails>
+      </Accordion>
+
+      <Accordion>
+        <AccordionSummary expandIcon={<ExpandMoreIcon />}>
+          <SectionTitle>{t.sectionLanguage}</SectionTitle>
+        </AccordionSummary>
+        <AccordionDetails>
+          <FormControl fullWidth>
+            <InputLabel id="mobile-language-select-label">{t.language}</InputLabel>
+            <Select
+              labelId="mobile-language-select-label"
+              value={preferences.language}
+              label={t.language}
+              // eslint-disable-next-line @typescript-eslint/no-explicit-any
+              onChange={(e) => setLanguage(e.target.value as any)}
```

Comment:
Same issue as above: using an any cast for the mobile Select onChange handler.

**Sugerencia:**
```typescript
Use SelectChangeEvent (typed) and narrow e.target.value to Locale instead of casting to any.
```